frontend/src/metabase/visualizations/lib/apply_tooltips.js

```diff
@@ -331,16 +331,12 @@ export const getStackedTooltipModel = (
   }));
 
   const hoveredSeries = seriesWithGroupedData[hoveredIndex];
-  const hoveredCardId = hoveredSeries?.card?.id;
-  const hoveredCardSeries = seriesWithGroupedData.filter(
-    series => series.card?.id === hoveredCardId,
-  );
-  const hasBreakout = hoveredCardSeries?.some(
+  const hasBreakout = seriesWithGroupedData.some(
```

Comment:
There can be series without breakouts and just with multiple metrics. I believe we should not include them in the calculation of percentages. Here is the [spec](https://www.notion.so/metabase/Offer-automated-percentages-on-stacked-charts-a8af42d2b81e458b827b679dac2b61c6?pvs=4#d0fc4c7acd114e3f9f75fae629905dbf)

Here is an example of a question with multiple series because of multiple metrics:
<img width="1215" alt="Screen Shot 2023-06-29 at 10 14 10 PM" src="https://github.com/metabase/metabase/assets/14301985/efeb8ad5-6bde-4d75-92af-c67978c55cd9">

The result of combining it with a question with breakouts:
<img width="852" alt="Screen Shot 2023-06-29 at 10 15 23 PM" src="https://github.com/metabase/metabase/assets/14301985/5212a6ca-09c9-4984-ae69-f14fade5748b">

So `seriesToShow` should only include series with breakouts

Reply:
I guess when adding a card with multiple metrics to a card with breakouts, they shouldn’t even be rendered together as a stacked bar chart, but I suppose we can just fix the tooltip first.

Reply:
Can you detail on how to reproduce this?  I created the chart with multiple metrics, but couldn’t merge it onto the stacked bar chart— the chart wasn’t listed on the right:

<img width="1043" alt="CleanShot 2023-07-05 at 14 46 43@2x" src="https://github.com/metabase/metabase/assets/116838/b61747df-60c2-4943-8c54-69bda6c9904d">

And likewise, I couldn’t merge the multi-metric chart onto the stacked bar chart (wasn’t listed on the right):

<img width="1044" alt="CleanShot 2023-07-05 at 14 46 17@2x" src="https://github.com/metabase/metabase/assets/116838/c9b56ac1-7be0-43ea-9ff9-a5bf3421e9be">

Reply:
I'm still able to reproduce it, please have a look:
<video src="https://github.com/metabase/metabase/assets/14301985/d90dcaa6-11f8-40c2-9b7e-16ec0ad28cb4" />

Reply:
Resolved the issue in this [slack discussion with Conor](https://metaboat.slack.com/archives/C02H619CJ8K/p1688594596966999?thread_ts=1687916296.041109&cid=C02H619CJ8K).  We’re erroneously only displaying percentages for breakouts not metrics, but stacked charts should always display percentages.